README.md

```diff
@@ -482,7 +482,7 @@ data:
 
 ## Troubleshooting and Support
 
-  - **I need to create a service binding to access a service instance that's in the `Delete Failed` state so that I can edit it to be able to delete it but the very state prevents me from doing that.**
+  - **I need to create a service binding to access a service instance that's in the `Delete Failed` state, but this state prevents me from doing that.**
  
```

Comment:
I would add some background, otherwise it seems that it is a feature. it is in the troubleshooting section.. but what exactly I have to troubleshoot...?
We have to emphasize that this is a 'recovery' mean for very specific cases.

@danielforsap you can use the background from the BLI https://jtrack.wdf.sap.corp/browse/SAPCPCFS-27546
but to make is much shorter.

proposal:
The deletion of my service instance failed, in order to fix the failure I have to create service binding, but this is not possible instance that's in the `Delete Failed` state.
(e.g. master xsuaa instance cannot be deleted since clone instances still exist, new binding has to be created in order to cleanup the clones)

Reply:
@avilupu I can present the case and then offer a solution instead of formatting it as Q&A.

Reply:
How about something like:
Probelm: A service instance is stuck in status "deletion failed", a binding is needed but it is not possible to create it in the cluster.
Solution: If instance is in delete failed state, it is possible to create a binding using smctl or sm API by adding a query param ....


here is a link to one of the bugs regarding this issue maybe it will give you the user's perspective
https://jtrack.wdf.sap.corp/browse/NGPBUG-191807?jql=text%20~%20%22There%20still%20exist%20clone%20instances%20for%20this%22

Reply:
Hi @kerenlahav,

Avi and I have already discussed this in a short call, the content has been updated.